proto/opamp.proto

```diff
@@ -802,6 +802,22 @@ message PackageStatus {
 
     // Error message if the status is erroneous.
     string error_message = 7;
+
+    // Optional details that may be of interest to a user.
+    // Should only be set if status is Downloading.
+    // Status: [Development]
+    PackageDownloadDetails download_details = 8;
```

Comment:
Please add this to the specification.md: https://github.com/open-telemetry/opamp-spec/blob/main/specification.md#packagestatus-message

We may need to rethink the need to duplicate this information, but for now it exists in 2 places: in the proto and in the specification.md.

Reply:
OK, added my changes; also fixed status to include the `INSTALL_PENDING` status that was missing